src/utils/router.js

```diff
@@ -0,0 +1,48 @@
+import { isPlainObject } from './object'
+
+export const isRouterLink = tag => tag !== 'a'
+
+export const computeTag = (props, parent) => {
+  return parent.$router && props.to && !props.disabled
```

Comment:
We should make a note that `parent` implies a functional component.  But `parent` could also be a `this` reference from a regular component.  So maybe renaming the variable to `parentOrThis` might be prudent.

Reply:
What about `context`?

Reply:
`parent` comes from the functional component `context` object... so using the var name `context` in this situation might be confusing.  And, non functional components don't have a `context`